content/docs/guides/verify-jwt.md

```diff
@@ -0,0 +1,169 @@
+| withHttps | Prepends the URL with the `https://` protocol. |
+| signOut | Signs user out and redirects them with the `/.pomerium/sign_out` endpoint. |
+
+## Manual verification
+
+Though you will likely verify signed headers programmatically in your application's middleware with a third-party JWT library, if you are new to JWT it may be helpful to show what manual verification looks like.
+
+1. Provide Pomerium with a base64-encoded Elliptic Curve ([NIST P-256](https://csrc.nist.gov/csrc/media/events/)) Private Key. In production, you'd likely want to get these from your key management service (KMS).
+
+   ```bash
+   openssl ecparam -genkey -name prime256v1 -noout -out ec_private.pem
+   openssl ec -in ec_private.pem -pubout -out ec_public.pem
+   # careful! this will output your private key in terminal
+   cat ec_private.pem | base64
+   ```
+
+   Copy the base64 encoded value of your private key to Pomerium's environmental configuration variable `SIGNING_KEY`.
+
+   ```bash
+   SIGNING_KEY=LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tCk1JR0hBZ0VBTUJNR0J5cUdTTTQ5QWdFR0NDcUdTTTQ5QXdFSEJHMHdhd0lCQVFRZzBQdk1VeVZGeWxTbGZ3eDgKSDBxMUVyOHZlOXBnY3ZzNkV6ZnR5OHlxNnFLaFJBTkNBQVM5ZC96TC9aSXd5ZGQ1RXZMb0xGMytHblVIUS9wdQpQaU45NDV1Y1RpTFRqMDhZalo3U0NJV2JHc2tiK0RIMzJ2aUc2KzRnb0FvWlFUM1R6b2kzRVl6OAotLS0tLUVORCBQUklWQVRFIEtFWS0tLS0tCg==
+   ```
+
+1. Reload Pomerium. Navigate to httpbin (by default, `https://httpbin.corp.${YOUR-DOMAIN}.com`), and log in as usual. Click **request inspection**. Select `/headers`. Click **try it out** and then **execute**. You should see something like the following.
+
+   ![httpbin displaying jwt headers](./img/jwt/inspect-headers.png)
+
+1. `X-Pomerium-Jwt-Assertion` is the signature value. It's less scary than it looks, and is basically just a compressed, JSON blob as described above. Navigate to [jwt.io], which provides a helpful user interface to manually verify JWT values.
+
+1. Paste the value of `X-Pomerium-Jwt-Assertion` header token into the `Encoded` form. You should notice that the decoded values look much more familiar.
+
+   ![httpbin displaying decoded jwt](./img/jwt/verifying-headers-1.png)
+
+1. Finally, we want to cryptographically verify the validity of the token. To do this, we will need the signer's public key. You can simply copy and paste the output of `cat ec_public.pem`.
+
+   ![httpbin displaying verified jwt](./img/jwt/jwt-payload.png)
+
+**Voila!** Hopefully walking through a manual verification has helped give you a better feel for how signed JWT tokens are used as a secondary validation mechanism in pomerium.
+
+:::caution
+
+In an actual client, you'll want to ensure that all the other claims values are valid (like expiration, issuer, audience and so on) in the context of your application. You'll also want to make sure you have a safe and reliable mechanism for distributing the public signing key to client apps (typically, a [key management service]).
+
```

Comment:
see above comment wrt to having a single source of validation checklist.

Reply:
I'm removing this, as this is covered already in the guide.